src/helpers/promise.js

```diff
@@ -1,4 +1,5 @@
 /* eslint no-proto:0 */
+/* eslint no-use-before-define:0 */
```

Comment:
I have tried to put ```globals``` option into ```.eslintrc``` and on top of the file. But I still having the problem.

```bash
error  'PromiseExt' was used before it was defined  no-use-before-define
```

Only the option no-use-before-define: 0 work.